otter/integration/tests/test_convergence.py

```diff
@@ -401,9 +403,8 @@ def test_scale_down_after_oobd_non_constrained_z_equal_y(self):
     def _scale_down_after_oobd_non_constrained_param(
             self, rcs, min_servers=0, max_servers=25, set_to_servers=0,
             oobd_servers=0, scale_servers=1):
-
         # This only applies if not constrained by max/min
-        converged_servers = set_to_servers + scale_servers
+        converged_servers = set_to_servers
```

Comment:
I think this change breaks the previous 3 tests (`test_scale_down_after_oobd_non_constrained_z_lessthan_y`, `test_scale_down_after_oobd_non_constrained_z_greaterthan_y`, `test_scale_down_after_oobd_non_constrained_z_equal_y`), because the scaling policy that triggers convergence *does* get executed.